src/modules/store/templates/paginated-products.tsx

```diff
@@ -66,14 +56,21 @@ export default async function PaginatedProducts({
     return null
   }
 
-  const {
+  let {
     response: { products, count },
   } = await getProductsList({
     pageParam: page,
     queryParams,
     countryCode,
   })
 
+  if (sortBy && ["price_asc", "price_desc"].includes(sortBy)) {
+    products = sortProductsByPrice(
```

Comment:
The issue here is that we are sorting just 1 page.
I think this will cause more issues and people will complain.

Reply:
In the V1 starter I think we grabbed 100 products and then handled pagination and sorting on the Nextjs server. 

We could do smth like that, or remove it entirely. Just feels a bit barebones to have a store page without basic sorting.